schema/checks.hcl

```diff
@@ -206,3 +206,57 @@ table "checks" {
     columns = [column.canary_id, column.type, column.name]
   }
 }
+
+table "check_statuses_1h" {
+  schema = schema.public
```

Comment:
needs a check_id, + rename interval_start to created_at, and `failed_checks` to `failed` and `successful_checks` to `passed`